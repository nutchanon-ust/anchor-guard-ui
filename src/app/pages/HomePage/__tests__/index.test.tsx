import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { HomePage } from '..';

const shallowRenderer = createRenderer();

describe('<HomePage />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<HomePage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
