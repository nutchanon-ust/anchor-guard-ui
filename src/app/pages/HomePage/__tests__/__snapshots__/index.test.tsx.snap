// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render and match the snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
    prioritizeSeoTags={false}
  >
    <title>
      Home Page
    </title>
    <meta
      content="A React Boilerplate application homepage"
      name="description"
    />
  </Helmet>
  <NavBar />
  <styled.div>
    <Masthead />
    <Features />
  </styled.div>
</React.Fragment>
`;
